templates/sushi_script.erb

```diff
@@ -40,4 +40,4 @@ fi
 chmod +x "$SAKE_PATH"
 
 # Preserves env
-sudo -E -u www-data "$SAKE_PATH" "$@"
+sudo -E -u www-data PHP_INI_SCAN_DIR=:<%= @cli_root %> "$SAKE_PATH" "$@"
```

Comment:
Using `PHP_IN_SCAN_DIR` will force PHP to look in this directory for `.ini` files INSTEAD of the default directory.

https://www.php.net/manual/en/configuration.file.php#configuration.file.scan

If you wanted to go down this route, then you may need to have an entire `php.ini` file (with all parameters).

Reply:
So far it only overrides because using the `:` before the cli root includes the system default ones as well as the overrides.

Reply:
> It is possible to scan multiple directories by separating them with the platform-specific path separator (; on Windows, NetWare and RISC OS; : on all other platforms; the value PHP is using is available as the PATH_SEPARATOR constant)

the docs make that reasonably clear IMO

Reply:
You are correct, I missed that in the examples at the bottom.